granulate_utils/config_feeder/core/utils.py

```diff
@@ -0,0 +1,13 @@
+import json
+from hashlib import blake2b
+from typing import Any, Dict, List, Union
+
+
+def get_config_hash(s: Union[str, Dict[str, Any], List[Any]], is_sorted: bool = False) -> str:
+    if isinstance(s, str):
+        s = s if is_sorted else json.dumps(json.loads(s), sort_keys=True)
+    else:
+        s = json.dumps(s, sort_keys=True)
+    h = blake2b(digest_size=10)
```

Comment:
Let's use a power of 2 number - I suggest 16,
@Jongy I think it's ok to use BLACK2B, WDYT?

Reply:
Unless performance is a key factor here, I see no reason to use a hash algorithm that is not well-known. If this code isn't called >1k times a second, use sha1 please

Reply:
blake2 has been around for more than 10 years. sha1 is legacy and not recommended anymore. even linux kernel replaced sha1 with blake. are you sure want to stick with sha1?

Reply:
Hmm, I suggested sha1 because it's more well known and personally I understand how to use it properly without additional configuration like done here. The fact that sha1 is considered unsafe is not an issue because the hash is only a unique identifier.

Use either, I don't mind. You can keep blake2 if you understand that its properties are useful here